src/redux/books/books.js

```diff
@@ -0,0 +1,41 @@
+const ADD_BOOK = 'book-store-react/books/ADD_BOOK';
+const REMOVE_BOOK = 'book-store-react/books/REMOVE_BOOK';
+// Reducer
+export const booksReducer = (state = [], action) => {
```

Comment:
- [ ] I kindly suggest that you export `booksReducer` as default because that is a project requirement.

## EXAMPLE
```
export default function ReactApp() {
  return <p>Exported Using Default Export</p>
}
```